keps/sig-node/2033-kubelet-in-userns-aka-rootless/README.md

```diff
@@ -654,67 +818,211 @@ Note that this feature gate does not support separating kubelet's user namespace
+
+Rollback: this question is not applicable. Rolling back requires recreating a new node instance.
 
-This section will be fulfilled when targeting beta graduation to a release.
+###### What specific metrics should inform a rollback?
+
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
+
+CrashLoopBackOffs
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+This question is not applicable. Rolling out and rolling back requires recreating a new node instance.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
+
+No
 
 ### Monitoring Requirements
 
-_This section must be completed when targeting beta graduation to a release._
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
 
-* **How can an operator determine if the feature is in use by workloads?**
+###### How can an operator determine if the feature is in use by workloads?
 
-N/A
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
 
-* **What are the SLIs (Service Level Indicators) an operator can use to determine 
-the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [X] Other (treat as last resort)
-    - Details: Use `systemctl --user is-system-running` to verify whether the processes (RootlessKit, kubelet, kube-proxy, and CRI) are running.
+They can determine if a Pod is running on a node that is running in UserNS.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### How can someone using this feature know that it is working for their instance?
 
-N/A
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
+- [X] Events
+  - Event Reason:  No CrashLoopBackOff
```

Comment:
This only gives them information this is working correctly. The question is how to know that in my cluster I have kubelets running in rootless mode? See my other question around whether there should be a metric stating that fact? Or what other information we can provide to the admin about that fact.

Reply:
> The question is how to know that in my cluster I have kubelets running in rootless mode?

Answered in 
https://github.com/kubernetes/enhancements/pull/5388#discussion_r2405365875

Reply:
Right, but the PR comment isn't sufficient. Please make sure to add that answer in the doc. It's perfectly reasonable to point to other places in the document, that are describing these bits.